.github/workflows/lint.yaml

```diff
@@ -11,13 +11,14 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - uses: conda-incubator/setup-miniconda@v2
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
 
       - name: Install packages
         run: |
-          conda env create --name test-env -f environment-dev.yml
+          pip install pre-commit
```

Comment:
is there a way to cache this pre-commit installation?

Reply:
I tried a few things, I stopped short of implementing: https://tobiasmcnulty.com/posts/caching-pre-commit/
Mostly because this takes 30s, and really is just a set of markdown files.